test/env.py

```diff
@@ -10,10 +10,11 @@
 HTTP_SERVER_FILE = os.path.join(RESOURCES_DIR, "testserver", "testserver.py")
 SRC_DIR = os.path.join(ROOT_DIR, "..", "src")
 
-TRAVIS = os.environ.get("TRAVIS", None)
-TRAVIS_JOB_NUMBER = os.environ.get("TRAVIS_JOB_NUMBER")
-SAUCE_USERNAME = os.environ.get("SAUCE_USERNAME")
-SAUCE_ACCESS_KEY = os.environ.get("SAUCE_ACCESS_KEY")
+TRAVIS = os.environ.get("TRAVIS", False)
+TRAVIS_EVENT_TYPE = os.environ.get("TRAVIS_EVENT_TYPE", None)
+TRAVIS_JOB_NUMBER = os.environ.get("TRAVIS_JOB_NUMBER", "localtunnel")
+SAUCE_USERNAME = os.environ.get("SAUCE_USERNAME", None)
+SAUCE_ACCESS_KEY = os.environ.get("SAUCE_ACCESS_KEY", None)
```

Comment:
If these are only used by `run_tests.py`, we could consider moving all this there and could probably remove the whole `env.py`. Not really related to this PR, though.